GameServerLib/Logic/GameObjects/Champion.cs

```diff
@@ -138,10 +141,8 @@ public int getTeamSize()
                     case "blue":
                         blueTeamSize++;
                         break;
-                    case "purple":
-                        purpTeamSize++;
-                        break;
                     default:
```

Comment:
I don't think we should assign the purple team on a default callback. This doesn't enforce proper format in the configuration file, and we're likely going to add the possibility for more teams at a later date too.